frontend/src/components/GroupCard/GroupCard.tsx

```diff
@@ -21,7 +21,7 @@ export const GroupCard = ({ groupObj, ...props }: GroupCardProps) => {
                     {groupObj.name}
                 </Typography>
                 <Typography variant="gray">Members:</Typography>
-                <Stack direction="row" spacing={1}>
+                <Stack direction="row" spacing={1} sx={{overflowX: 'auto', flexWrap: 'nowrap', paddingBottom: 2}}>
```

Comment:
Could you try to use a different component instead of `Stack` to address the issue, so that the chips wrap instead of needing to be scrolled?

 e.g., https://mui.com/material-ui/react-chip/#chip-array

Reply:
Added new commit, looks like this 
![image](https://github.com/user-attachments/assets/f0ae36df-1045-4f4f-9695-0bb55b41abce)

Reply:
super slay